src/plugins/intel_gna/src/gna_plugin.hpp

```diff
@@ -48,6 +49,7 @@ class GNAPlugin : public InferenceEngine::IInferencePlugin {
     std::shared_ptr<GnaInputs> inputs_ptr_;
     GnaOutputs outputs_;
     std::shared_ptr<GNAGraphCompiler> m_graph_compiler;
+    bool isAvx2Supported;
```

Comment:
I notivced that there are a lot of conventions used for variables names but I suggest the most frequest one :)